olive/model/handler/pytorch.py

```diff
@@ -58,9 +56,9 @@ def run_session(
         **kwargs: dict[str, Any],
     ) -> Any:
         if isinstance(inputs, dict):
-            results = session.generate(**inputs, **kwargs) if self.generative else session(**inputs, **kwargs)
+            results = session(**inputs, **kwargs)
         else:
-            results = session.generate(inputs, **kwargs) if self.generative else session(inputs, **kwargs)
+            results = session(inputs, **kwargs)
         return results
```

Comment:
With this change, aren't we dropping support for non-hf generative pytorch models?

Or, is the consideration that this code path gets used only for latency computation and so use of `forward` call over `generate` call is justified.